q2_fmt/_peds.py

```diff
@@ -236,6 +238,49 @@ def _compute_peds(peds_df: pd.Series, peds_type: str, peds_time: int,
     return peds_df
 
 
+# prep method
+def _rename_features(level_delimiter, data: pd.DataFrame):
+    if ("recipients with feature" in data.columns and
+            level_delimiter is not None):
+        group_name = data["group"].attrs['title']
+        subject_name = data["subject"].attrs['title']
+        measure_name = data["measure"].attrs['title']
+
+        y_labels = []
+        seen = Counter()
+        subject_seen = []
+        for i, sub in enumerate(data['subject']):
+            if level_delimiter in sub:
+                fields = [field for field in sub.split(level_delimiter)
+                          if not field.endswith('__')]
+            else:
+                # This is necessary to handle a case where the delimiter
+                # isn't found but the sub ends with __. In that case, sub would
```

Comment:
I think it would be good to add a test for this case as well, just to confirm behavior is as expected!

Reply:
Just to confirm you want a test case that ends with __ to see if it gets dropped?

Reply:
Yes exactly!